api/v1alpha1/jwt_types.go

```diff
@@ -108,6 +111,28 @@ type RemoteJWKS struct {
 	// +kubebuilder:validation:MinLength=1
 	// +kubebuilder:validation:MaxLength=253
 	URI string `json:"uri"`
+	// Duration after which the cached JWKS should be expired. If not specified, default cache duration is 5 minutes.
+
+	// +kubebuilder:default="300s"
+	// +kubebuilder:validation:Format=duration
+	// +optional
+	CacheDuration *metav1.Duration `json:"cacheDuration,omitempty"`
+
+	// Fetch Jwks asynchronously in the main thread before the listener is activated. Fetched Jwks can be used by all worker threads.
+	// +optional
+	AsyncFetch *JwksAsyncFetch `json:"asyncFetch,omitempty"`
```

Comment:
can we rm AsyncFetch from this API PR, since its already enabled by default 
prefer if we address retry semantics as part of https://github.com/envoyproxy/gateway/issues/6525
this should help make this PR move faster

Reply:
Ok, so if I understand it correctly, in this PR, we just want to add the CacheDuration field and not the AsyncFetch field right?

Reply:
yeah thanks